formfyxer/lit_explorer.py

```diff
@@ -972,6 +972,39 @@ def get_citations(text: str, tokenized_sentences: List[str]) -> List[str]:
     return citations_with_context
 
 
+def get_sensitive_fields(fields: List[str]) -> List[str]:
+    """
+    Given a list of fields, identify those related to sensitive information. Sensitive fields include 
+    Social Security Number(SSN)), Driver's License (DL), and account numbers.
+    """
+    # NOTE: omitting CID since it has a lot of false positives.
+    field_patterns = {
+        "Social Security Number": [
+            "social[\W_]*security[\W_]*number",
+            "SSN",
+            "TIN$"
+        ],
+        "Bank Account Number": [
+            "account[\W_]*number",
+            "ABA$",
+            "routing[\W_]*number",
+            "checking"
+        ],
+        "Credit Card Number": [
+            "credit[\W_]*card",
+            "(CV[CDV]2?|CCV|CSC)"
+        ],
+        "Driver's License Number": [
+            "drivers[\W_]*license",
+            ".?DL$"
+        ]
+    }
+    text = "\n".join(fields)
+    field_regexes = {name: re.compile("|".join(patterns), re.IGNORECASE | re.MULTILINE) for name, patterns in field_patterns.items()}
+    sensitive_fields = [name for name, regex in field_regexes.items() if re.search(regex, text)]
```

Comment:
We usually only use names with `fields` in them if they map directly to the form fields. Suggesting this name change everywhere, unless you change the return type

```suggestion
    sensitive_data_types = [name for name, regex in field_regexes.items() if re.search(regex, text)]
```